src/main/java/gov/usds/case_issues/services/KPIService.java

```diff
@@ -0,0 +1,84 @@
+package gov.usds.case_issues.services;
+
+import java.time.DayOfWeek;
+import java.time.ZonedDateTime;
+import java.time.temporal.TemporalAdjuster;
+import java.time.temporal.TemporalAdjusters;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.validation.annotation.Validated;
+
+import gov.usds.case_issues.db.repositories.BulkCaseRepository;
+import gov.usds.case_issues.services.model.CaseGroupInfo;
+import gov.usds.case_issues.validators.TagFragment;
+
+@Service
+@Validated
+public class KPIService {
+	private static final int WEEKS = 10;
+	private static TemporalAdjuster START_OF_WEEK = TemporalAdjusters.previousOrSame(DayOfWeek.MONDAY);
+
+	@Autowired
+	private BulkCaseRepository _bulkRepo;
+	@Autowired
+	private CaseListService _caseListService;
+
+	public Map<String, Object> getKPIData(
```

Comment:
Sometime soon it's probably worth turning this into a static object, so that we have predefined field names we can make consistent.